packages/client/src/components/MovieCard/MovieCard.jsx

```diff
@@ -1,35 +1,105 @@
+import * as React from 'react';
+import { Card, CardMedia, Typography, Rating, Box } from '@mui/material';
+import styled from '@emotion/styled';
 import FavoriteIcon from '@mui/icons-material/Favorite';
 import FavoriteBorderIcon from '@mui/icons-material/FavoriteBorder';
-import { Box, Card, CardMedia, Rating, Typography } from '@mui/material';
-import * as React from 'react';
 import { useFavorites } from './useFavorites'; // created a custom hook to manage favorites
 
 const BasicRating = () => {
-  return (
-    <Rating
-      name="read-only"
-      value={4}
-      readOnly
-      sx={{
-        position: 'relative',
-        left: '1.96rem',
-        borderRadius: '0.59',
-        maxHeight: 'min-content',
-        width: '3rem',
-        top: '2rem',
-        paddingBottom: '4rem',
-        color: '#003E2F',
-      }}
-    />
-  );
+  const StyledRating = styled(Rating)({
+    position: 'relative',
+    left: '1.96rem',
```

Comment:
To keep the positions consistent, it is a common practice to use multiples of 0.25 when using rem.

Reply:
yeah you are definitely right, i was trying to to do pixel perfect according to Figma design but i changed them :)